packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -14,11 +15,6 @@ import 'package:smooth_app/helpers/score_card_helper.dart';
 import 'package:smooth_ui_library/smooth_ui_library.dart';
 import 'package:smooth_ui_library/util/ui_helpers.dart';
 
-const List<String> _SCORE_ATTRIBUTE_IDS = <String>[
```

Comment:
Looks like this for me:
![Screenshot_1636989493](https://user-images.githubusercontent.com/7541730/141807162-2a94c921-435c-4260-b721-4ddf9d583fb4.png)

From the product spec's discussion I remember that it was decided to always show the two scores (Attribute and ecoscore) even if it isn't marked as mandatory by the user

Reply:
@jasmeet0817 The big ones would therefore be Nutriscore, Ecoscore, then all the mandatory attributes, right?
And the other ones would be all the other important/not important attributes, right?

If the tiny bug you're referring to is not about nutriscore and ecoscore, but the fact that in your screenshot there's the logo and no label, it's because I don't get any label for vegan in that case.

Reply:
so only the Nutri and Eco score should be displayed using `ScoreCard`, they are considered 1st class Attributes. Other attributes are all displayed in the same manner, by calling `_buildAttributeChipForValidAttributes`.